apps/prairielearn/src/pages/courseSyncs/courseSyncs.html.ts

```diff
@@ -57,6 +57,7 @@ export function CourseSyncs({
         </script>
         ${renderEjs(import.meta.url, "<%- include('../partials/navbar'); %>", resLocals)}
         <main id="content" class="container-fluid">
+          <h1 class="sr-only">Course Sync</h1>
```

Comment:
On pages like this where we have multiple cards, my inclination would be to use `<h2>` for card titles to give screen readers additional navigational landmarks. I won't block on this and I'd be happy to save this for another PR.

Reply:
Makes sense! I have opened issue #10446 and mentioned this as part of that issue.